tests/quick_tests/run.cmake

```diff
@@ -22,7 +22,7 @@ SEPARATE_ARGUMENTS(ALL_TESTS)
 EXECUTE_PROCESS(COMMAND ${CMAKE_CTEST_COMMAND} --force-new-ctest-process --output-on-failure -O quicktests.log RESULT_VARIABLE res_var)
 
 if(NOT "${res_var}" STREQUAL "0")
-  MESSAGE( "
+  MESSAGE(SEND_ERROR "
 
 *******************************     WARNING     *******************************
```

Comment:
Good point. We cannot do `MESSAGE(WARNING ...` because that will still return zero.